### ISSUE TITLE
PRC Format Table 7

### ISSUE THREAD

**Issue by mvrhel** [pdf-association/pdf-issues#447]
Table 7 describes the FileStructureHeader.  The last bit in the header describes the uncompressed files which are the image files used for textures.  The header includes file_count and an array of  UncompressedFile type.  This is not correct.  UncompressedFile is defined in Table 5 and is defined by a count and an array of size count of Uncompressed blocks.  To me it appears that the FileStructureHeader array is actually of type Uncompressed blocks (Table 11).

**Comment by t3dr0** [comment]
I agree, changing the Table 7 "files" entry type to "Array <UncompressedBlock>[file_count]" brings it into alignment with Table 12.

**Comment by stuartgalt** [comment]
![Image](https://github.com/user-attachments/assets/0abefdd2-0b9b-43d4-afcd-95a568a9e1ba)

**Comment by t3dr0** [comment]
Looks good, thank you!

**Comment by petervwyatt** [comment]
Implemented
